engine.py

```diff
@@ -39,8 +40,13 @@ def tank_mobu_exception_trap(ex_cls, ex, tb):
 
     # now output it    
     try:
-        from PySide import QtGui
```

Comment:
[nitpick] Should we also change references to `tank` over to `sgtk` while we're doing cleanup/modernization work in here?

Reply:
cool! will do!

Reply:
done!